webhook.rst

```diff
@@ -0,0 +1,132 @@
+Webhook
+=======
+
+.. versionadded:: 6.3
+
+    The Webhook component was introduced in Symfony 6.3
+
+The Webhook component is used to respond to remote webhooks to trigger actions
+in your application. This document focuses on using webhooks to listen to remote
+events in other Symfony components.
+
+Installation
+------------
+
+.. code-block:: terminal
+
+    $ composer require symfony/webhook
+
+Usage in combination with the Mailer component
+----------------------------------------------
+
+When using a third-party mailer, you can use the Webhook component to receive
+webhook calls from the third-party mailer.
+
+In this example Mailgun is used with ``'mailer_mailgun'`` as webhook type.
+Any type name can be used as long as it's unique. Make sure to use it in the
+routing configuration, the webhook URL and the Remote Event Consumer.
+
+Install the third party mailer as described in the documentation of the mailer
```

Comment:
We may can add a link to Mailer documentation

Reply:
Indeed. I also want to link from the Mailer documentation to the webhook as well.

Reply:
Can I link to the third party bit directly?

Reply:
As we don't have much info for a third party bridge in the docs we could link to the overview or link to the README in the code repo.

What would you prefer? 

Btw, WDYT about this PR? https://github.com/symfony/symfony-docs/pull/19263

Reply:
Ah, nice. Yes, that would be a good extra bit of information in a good spot.